### PR TITLE
Set up UI

### PR DESCRIPTION
**Pull request by KingsleyIbe**
In this milestone, I set up UI for the to-do-list-app project using webpack.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -26,11 +26,6 @@ const listItems = [
     description: "Join morning session call",
     completed: false,
     index: 4
-  },
```
Comment (carlos-ssh) [inline src/index.js:29]: Kindly add these required files for your linters setup:
- .eslintrc.json
- .hintrc
- .stylelintrc.json


You can have access to these files [here](https://github.com/microverseinc/linters-config/tree/master/html-css-js) or in your general requirements of your project you can have access to these files.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @KingsleyIbe , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KingsleyIbe** [comment]
Linter files have been added to the project as recommended by the last reviewer.

**Review by rindrajosia — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**